src/components/index/CommunityCard.tsx

```diff
@@ -102,7 +102,11 @@ const CommunityCard = ({
           alignItems="center"
         >
           <Img
-            src={`${imageUrl}`}
+            src={`${
```

Comment:
I don't think this is something, we should handle in the code, we could just replace in the .json, and write an issue in the social-token-scraper repo

Reply:
Oh, I actually solved it this way, because we are using the full size images on the `/[community]` page, so we'll need those too, that's why I left the large image URLs in the JSON file. 

Not sure what would be the best solution in this situation, because afaik we don't have multiple image sizes on our AWS S3, so there isn't a good solution for every image (AWS vs Coingecko) right now I guess?

Reply:
You are right, I forgot that we need them on the `/[community]` page as well. It still might be a bit better to have the small coingecko URL-s by default in the JSON, and replace it on the `/[community]` page, so when rendering the explorer, we wouldn't have the condition and the string operation for every card, we would only do it once in `/[community]`.
However, I'm not sure if this would have a noticable effect on performance.

Reply:
That's a good point, I'll do this, thanks!